### ISSUE TITLE
NVDA is not announcing correct information for "column1" it is announcing code at "Databases for assessment" page: A11y_[BM]SQL Migration extension for Azure Data Studio_Migrate to azure sql_Databases for assessment_Labels or Instructions

### ISSUE THREAD

**Issue by Raj-ghuge** [microsoft/azuredatastudio#25922]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev3; #BM_SQL Migration extension for Azure Data Studio_Win32_AUG2024; #SQL Migration extension for Azure Data Studio; #WCAG3.3.2; #Labels or Instructions; #FTP; #Win32;

## Environment Details:
Application Name: SQL Migration extension for Azure Data Studio
Microsoft Windows 11 version 23H2 (OS Build 22631.4037)
Version: 1.49.1 (user setup)
Commit: 2f696aa95a13ccabd7e0a4431d1d759e3a7b8cfd
Date: 2024-08-13T18:12:38.831Z
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
Azure SQL Migration v1.4.8
Screen Reader: NVDA (2024.2) and JAWS 2024

## Repro Steps:

1. Turn on NVDA.
2. Install Azure data studio and open.
3. Navigate by using TAB key till Extension press ENTER key and install "Azure SQL Migration"
4. Navigate by using TAB key till Respective server and right click.
5. Navigate by using TAB key till manage and press ENTER key.
6. Navigate by using TAB key till "Azure SQL Migration" and press ENTER key.
7. Navigate by using TAB key till migrate to azure sql and press ENTER key, saved session page will open.
8. Select start new session and give next, "Databases for assessment" page will open.
9. Navigate by using TAB key to the grid table at "Column 1" and observe the issue.

## Actual results:
NVDA is not announcing correct information for "column1" it is announcing code at "Databases for assessment" page.
 
## Expected results:
NVDA should announce correct information for "column1" and should not announce code at "Databases for assessment" page.

## User Impact:
The screen reader user will might face difficulties and won't be able to perform action on it.
